paf-mvp-benchmark/K6/README.md

```diff
@@ -0,0 +1,11 @@
+# OneKey Capacity Planning
+
+## Toolkit
+K6 load testing tools:\
+- [K6 Official Documentation](https://k6.io/docs/)
+- [A useful summary](https://pflb.us/blog/k6-user-manual/)
+
+K6_PROMETHEUS_REMOTE_URL=http://localhost:9090/api/v1/write
```

Comment:
Is it the command that should be ran?
If yes, can you please put it in ` ``` ` ?
```markdown
```shell
K6_PROMETHEUS_REMOTE_URL=http://localhost:9090/api/v1/write

k6 run .\scenarios.js -o output-prometheus-remote
```

Reply:
By scenarios.js do you mean to choose a scenario like `first-visit-with-3pc.js` for example? because there is no file named scenarios.js.  
The second argument `output-prometheus-remote` is also not clear for me. It seems that k6 is expecting one of this types: `available types are: cloud, csv, influxdb, json, statsd`

Reply:
The doc will be updated.